src/resources/help/en_US/relnotes3.2.0.html

```diff
@@ -58,7 +58,7 @@
   <li>Enhanced the MFEM reader so that it provides the original cell numbers that allow VisIt to successfully eliminate mesh lines that were not part of the original high order mesh.</li>
   <li>Enabled the Rect reader as an optional reader on Windows.</li>
   <li>Enabled the ZipWrapper reader on Windows.</li>
-  <li>Upgraded the CGNS library used by the CGNS reader from 3.2.1 to 4.1.2. The new CGNS library includes a fix for the handling of RIND zones in structured grids, so the results may change when viewing that type of data.</li>
+  <li>Upgraded the CGNS library used by the CGNS reader from 3.2.1 to 4.1.2. The new CGNS library includes a fix for the handling of RIND zones in structured grids, so the results may change when viewing that type of data.
```

Comment:
The line is missing the </li> terminator.

Reply:
Thanks, sorry about that.